includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -300,6 +300,7 @@ public function pull( $items ) {
 
 			$post_props = get_object_vars( $post );
 			$post_array = array();
+			$original_post_parent = wp_get_post_parent_id( $item_array['post_id'] );
```

Comment:
I _think_ this is not referring to the correct post in this context. The original post is the `remote_post_id`. The WP_Post object for this post is stored in the `$post` variable on line 294, so we could use `$post->post_parent`.